### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by dyladan**
:robot: I have created a release \*beep\* \*boop\*
---
<details><summary>@opentelemetry/browser-extension-autoinjection: 0.27.4</summary>

### Bug Fixes

* **browser-extension-autoinjection:** update eslint-plugin-json5 version to 0.1.4 ([#896](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/896)) ([e47fcaa](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/e47fcaa1a04de35b096f373f44abbf87ff4125f2))
</details>
<details><summary>@opentelemetry/instrumentation-aws-sdk: 0.5.1</summary>

### Bug Fixes

* **opentelemetry-instrumentation-aws-sdk:** error when ReturnConsumedCapacity is set to None ([#899](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/899)) ([e7ab4d0](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/e7ab4d056b6663f593b47af7c3e8014a72a963fe))
</details>
<details><summary>@opentelemetry/instrumentation-fastify: 0.25.1</summary>

### Bug Fixes

* typo in fastify description ([#891](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/891)) ([adbd6dc](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/adbd6dcb0af6540a6d10b7e2ceaaf2c69a3e1146))
</details>
<details><summary>@opentelemetry/instrumentation-graphql: 0.27.4</summary>

### Bug Fixes

* **graphql:** fix `graphql.operation.name` field ([#903](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/903)) ([5529261](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/552926146c838efd7e2b778ae6fb815e9e304965))
</details>
<details><summary>@opentelemetry/instrumentation-mongodb: 0.29.0</summary>

### Features

* **mongodb4:** added mongodb4 instrumentation ([#869](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/869)) ([47700e1](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/47700e10dc6e4bd9ba0255cae85dec07ab4dd448))
</details>
<details><summary>@opentelemetry/instrumentation-mysql2: 0.29.0</summary>

### Features

* add mysql2 responsehook ([#915](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/915)) ([f436601](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/f4366015e2a628efc4cb8a47d508ba5620175f88))
* support mysql2 v1 ([#908](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/908)) ([d3883d3](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/d3883d38b7cf929404cf4eac9a9a48b7d1f4327f))
</details>
<details><summary>@opentelemetry/instrumentation-redis: 0.28.1</summary>

### Bug Fixes

* **opentelemetry-instrumentation-redis:** add condition before error print ([#897](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/897)) ([f1d2fd0](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/f1d2fd084c8e3e494e3606c4eca53158495f43f6))
</details>
<details><summary>@opentelemetry/instrumentation-winston: 0.27.2</summary>

### Bug Fixes

* **documentation:** mixed up winston readme for hook ([#910](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/910)) ([0a557e8](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/0a557e86c0d70dc6ab7af9736b1746f89c5fc8bf))
</details>
<details><summary>@opentelemetry/instrumentation-long-task: 0.1.0</summary>

### Features

* **longtasks:** allow callback to add span attributes on collection ([#863](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/863)) ([1f68004](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/1f68004ef9b25b0d260159f4b1e2f279b1a64649))
</details>
<details><summary>@opentelemetry/auto-instrumentations-node: 0.27.4</summary>

### Dependencies

* The following workspace dependencies were updated
  * dependencies
    * @opentelemetry/instrumentation-aws-sdk bumped from ^0.5.0 to ^0.5.1
    * @opentelemetry/instrumentation-fastify bumped from ^0.25.0 to ^0.25.1
    * @opentelemetry/instrumentation-graphql bumped from ^0.27.3 to ^0.27.4
    * @opentelemetry/instrumentation-mongodb bumped from ^0.28.0 to ^0.29.0
    * @opentelemetry/instrumentation-mysql2 bumped from ^0.28.0 to ^0.29.0
    * @opentelemetry/instrumentation-redis bumped from ^0.28.0 to ^0.28.1
    * @opentelemetry/instrumentation-winston bumped from ^0.27.0 to ^0.27.2
</details>

This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

**Comment by Flarna** [comment]
I miss that ones added in #873

File: .release-please-manifest.json [hunk 1]
```diff
@@ -1 +1 @@
...
...try-instrumentation-koa":"0.28.1","plugins/node/opentelemetry-instrumentation-memcached":"0.27.1","plugins/node/opentelemetry-instrumentation-mongodb":"0.29.0","plugins/node/opentelemetry-instrumentation-mysql":"0.27.1","plugins/node/opentelemetry-instrumentation-mysql2":"0.28.0","plugins/node/opentelemetry-instrumentation-nestjs-core":"0.28.3","plugins/node/opentelemetry-instrumentation-net":"0.27.1","plugins/node/opentelemetry-instrumentation-pg":"0.28.0","plugins/node/opentelemetry-instrumentation-pino":"0.28.1","plugins/node/opentelemetry-instrumentation-redis":"0.28.0","plugins/node/opentelemetry-instrumentation-restify":"0.27.2","plugins/node/opentelemetry-instrumentation-router":"0.27.1","plugins/node/opentelemetry-instrumentation-winston":"0.27.1","plugins/web/opentelemetry-instrumentation-document-load":"0.27.1","plugins/web/opentelemetry-instrumentation-user-interaction":"0.28.1","plugins/web/opentelemetry-plugin-react-load":"0.26.1","propagators/opentelemetry-propagator-aws-xray":"1.0.1","propagators/opentelemetry-propagator-grpc-census-binary":"0.25.1","propagators/opentelemetry-propagator-ot-trace":"0.25.1","plugins/node/opentelemetry-instrumentation-fastify":"0.25.0"}
```
Comment (vmarchaud) [inline .release-please-manifest.json:1]: i've manually fixed the conflict that https://github.com/open-telemetry/opentelemetry-js-contrib/commit/9f3b22774cad0721f9cbb18dbb9628bbf2e76982 introduced, if someone review it please check that i didn't make any mistake but it seems ok to me

**Comment by Flarna** [comment]
lucky me I found a typo in fastify instrumentation and created a fix PR. Finally it got included by release please.

Not sure if a patch increment is correct here. Besides that I think that changelog misses e.g. #769 (which is a feat commit resulting in a minor update).

**Comment by dyladan** [comment]
> lucky me I found a typo in fastify instrumentation and created a fix PR. Finally it got included by release please.
> 
> Not sure if a patch increment is correct here. Besides that I think that changelog misses e.g. #769 (which is a feat commit resulting in a minor update).

I will update it manually before merging this

**Review by Rauno56 — APPROVED** [review]

**Comment by dyladan** [comment]
This is missing fastify, mysql2, and ioredis changes because incorrect commit messages were used

#919 updates release-please so that the commit override feature works. Tested locally

**Pull request closed without merging**